lib/config-validator.ts

```diff
@@ -78,21 +88,23 @@ type PackageJson = {
     // ignore
   }
   try {
-    const fileConfig = getConfig(process.env);
-    console.log(`Validating config.js`);
-    try {
-      await validate('config.js', fileConfig);
-    } catch (err) {
-      console.log(`config.js is not valid Renovate config`);
-      returnVal = 1;
+    const fileConfig = getFileConfig(process.env);
+    if (!equal(fileConfig, {})) {
+      logger.info(`Validating config.js`);
```

Comment:
log `process.env.RENOVATE_CONFIG_FILE` if available?
```suggestion
      logger.info(`Validating ${process.env.RENOVATE_CONFIG_FILE ?? 'config.js'}`);
```